Sources/NIOPosix/System.swift

```diff
@@ -249,6 +265,91 @@ internal func syscall<T: FixedWidthInteger>(blocking: Bool,
     }
 }
 
+@inline(__always)
+@discardableResult
+internal func syscall<T>(where function: String = #function,
```

Comment:
this helper and `syscall` variant that operates on the `OpaquePointer` seem to be platform-specific and used once (at least for now), do you think it's worth inlining them or making them plaftorm-specific?

Reply:
I think it's fine to keep them separate. It makes it easier to follow the general pattern.